src/components/ViewSpecRegistrationModal.vue

```diff
@@ -255,8 +276,13 @@ export default defineComponent({
       const { pageSize, page: pageNumber } = payload
 
       const requestOptions = {
-        productId: props.product?.id || $route.params.product,
-        productVersionId: props.version?.id || $route.params.product_version,
+        ...(useAppRegV2
+          ? {
+              filterAuthStrategyId: authStrategyId.value
+            }
+          : {}),
```

Comment:
For some reason this doesn't actually add the correct value.

This should work:

```suggestion
...(useAppRegV2 && { filterAuthStrategyId: authStrategyId.value }),
```